src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,9 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from "../CommentInfo/CommentInfo";
+
+export const CommentList = ({ comments }) => (
```

Comment:
Props type is not defined for the CommentList component. It's good practice to define prop types in React to ensure the correct data type is passed to components.